addons/logistics_wirecutter/CfgVehicles.hpp

```diff
@@ -8,19 +8,76 @@ class CfgVehicles {
 
     class Wall_F;
     class NonStrategic;
-
-    class Land_Net_Fence_4m_F: Wall_F { GVAR(isFence) = 1; };
-    class Land_Net_Fence_8m_F: Wall_F { GVAR(isFence) = 1; };
-    class Land_New_WiredFence_5m_F: Wall_F { GVAR(isFence) = 1; };
-    class Land_New_WiredFence_10m_F: Wall_F { GVAR(isFence) = 1; };
-    class Land_SportGround_fence_F: Wall_F { GVAR(isFence) = 1; };
-    class Land_Wired_Fence_4m_F: Wall_F { GVAR(isFence) = 1; };
-    class Land_Wired_Fence_8m_F: Wall_F { GVAR(isFence) = 1; };
-    class Land_SportGround_fence_noLC_F: Wall_F { GVAR(isFence) = 1; };
-    class Land_Mil_WiredFence_F: Wall_F { GVAR(isFence) = 1; };
-    class Land_IndFnc_Corner_F: Wall_F { GVAR(isFence) = 1; };
-    class Land_IndFnc_9_F: Wall_F { GVAR(isFence) = 1; };
-    class Land_IndFnc_3_Hole_F: Wall_F { GVAR(isFence) = 1; };
-    class Land_IndFnc_3_F: Wall_F { GVAR(isFence) = 1; };
-    class Land_Razorwire_F: NonStrategic { GVAR(isFence) = 1; };
+    class Land_Net_Fence_4m_F: Wall_F {
```

Comment:
it didn't break Coding Guidelines before and was quite compact

Reply:
I prefer the spread out variant more as it looks cleaner. I would however prefer a Macro much more.

Reply:
What about `MACRO_FENCE(Land_Net_Fence_4m_F,Wall_F);`?

Reply:
I tried it out and it is less readable than having them spread out.